common/src/app/common/files/migrations.cljc

```diff
@@ -1552,6 +1552,25 @@
 
     (update data :pages-index d/update-vals update-page)))
 
+(defmethod migrate-data "0010-fix-swap-slots-pointing-non-existent-shapes"
+  [data _]
+  (letfn [(fix-shape [page shape]
+            (let [shape-swap-slot (ctk/get-swap-slot shape)]
+              (if shape-swap-slot
```

Comment:
this let + if can be replaced with `if-let [...`

Reply:
Better, I can remove the let completly